src/components/Layout/NavBar.component.jsx

```diff
@@ -63,17 +64,22 @@ const useStyles = makeStyles((theme) => ({
 
 const Navbar = () => {
   const classes = useStyles();
+  const globalContext = useGlobal();
   const [isChecked, setIsChecked] = useState(false);
-  const [searchText, setSearchText] = useState('wizeline');
-  const searchContext = useContext(SearchContext);
+  const [innerSearchText, setInnerSearchText] = useState('wizeline');
```

Comment:
Extract the value for the input from the state instead of having a hardcoded value here. If you want to have a default when the app loads, add it to the initial state on your reducer.